src/main/java/com/autotune/Autotune.java

```diff
@@ -88,6 +96,10 @@ public static void main(String[] args) {
             e.printStackTrace();
             System.exit(1);
         }
+
+        // Read and execute the DDLs here
+        executeDDLs();
```

Comment:
add try and catch

and jetty server should not start if major exception other than (adding unique constraint)

Reply:
Done